hardware_enablement/kmm-release-notes.adoc

```diff
@@ -28,4 +27,297 @@ toc::[]
+
+** *Result:* The KMM Operator Hub environment works as expected.
+
+// MGMT-20852
+* The KMM Operator Hub environment cannot build when finalisers can not be set on a resource.
+
+** *Cause*: The Kernel Module Management (KMM) Operator displays an error with the 'ManagedClusterModule' controller failing to build. This is due to the missing 'ModuleImagesConfig' (MIC) resource finalizers and Role-based Action Control (RBAC) permissions for the KMM Operator Hub environment.
+
+** *Consequence*: The KMM Operator Hub environment cannot build images.
+
+** *Fix*: The RBAC permissions are updated to allow updating finalizers on the MIC resource, and then the appropriate rules created.
+
+** *Result:* The KMM Operator Hub environment builds images without errors with the 'ManagedClusterModule' controller.
+
+// MGMT-20861
+* The 'PreflightValidationOCP' custom resource, with a `kernelVersion: tesdt` causes the KMM Operator to panic.
+
+** *Cause*: Creating a `PreflightValidationOCP` custom resource (CR), with a 'kernelVersion' flag that is set to 'tesdt', causes the Kernel Module Management (KMM) Operator to generate a panic runtime error.
+
+** *Consequence*: Entering invalid kernel versions causes the KMM Operator to panic.
+
+** *Fix*: A webhook - a method for one application to automatically send real-time data to another application when a specific event occurs - has been added to the `PreflightValidationOCP` CR.
+
+** *Result:* The `PreflightValidationOCP` CR with invalid kernel versions can no longer be applied to the cluster, therefore, preventing the operator from generating a panic runtime error.
+
+// MGMT-20866
+* The `PreFflightValidationOCP` custom resource, with a `kernelVersion` flag that is different that the one of the cluster, does not work.
+
+** *Cause*: Creating a `PreflightValidationOCP` custom resource (CR), with a 'kernelVersion' flag that is different from the one of the cluster, does not work.
+
+** *Consequence*: The Kernel Module Management (KMM) Operator is unable to find the Driver Toolkit (DTK) input image for the new kernel version.
+
+** *Fix*: Users should use the `PreflightValidationOCP` CR and explicitly set the 'dtkImage' field in the CR.
+
+** *Result:* Using the fields `kernelVersion` and `dtkImage` the feature can build installed modules for target {product-title} versions.
+
+// MGMT-20888
+* The KMM Operator version 2.4 documentation is updated with `PreflightValidationOCP` information.
+
+** *Cause*: Previously, when creating an `PreflightValidationOCP` CR, you were required to supply the release-image. This has now changed and you need to set the `kernelVersion` the `dtkImage` fields.
+
+** *Consequence*: The documentation was outdated and required an update.
+
+** *Fix*: The documentation is updated with the new support details.
+
+** *Result:* The KMM preflight feature is documented as expected.
+
+// MGMT-20892
```

Comment:
Looks like a duplicate of this issue [here](https://github.com/openshift/openshift-docs/pull/95248/files#diff-a508e43343db975c5b8b7e3b4f30ef90441b9420f08df906454a6d85aa693514R89)

Reply:
You are correct. Deleted this instance. Thx! Good call!